tests/unittest/_torch/visual_gen/test_wan21_i2v_teacache.py

```
@@ -256,5 +256,5 @@ def test_wan22_raises_if_teacache_enabled(self):
             dtype="bfloat16",
             cache=TeaCacheConfig(),
         )
-        with pytest.raises(ValueError, match="TeaCache is not supported for Wan 2\\.2"):
+        with pytest.raises(ValueError, match="Wan 2.2 TeaCache requires explicit"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)